sw/device/lib/crypto/drivers/kmac.c

```diff
@@ -545,6 +549,14 @@ static status_t kmac_write_key_block(kmac_blinded_key_t *key) {
       KMAC_KEY_LEN_REG_RESVAL, KMAC_KEY_LEN_LEN_FIELD, key_len_enum);
   abs_mmio_write32(kKmacBaseAddr + KMAC_KEY_LEN_REG_OFFSET, key_len_reg);
 
+  // Write random words to the key registers.
+  for (size_t i = 0; i * sizeof(uint32_t) < key->len; i++) {
```

Comment:
In other places we are loading each key in a separate loop and checking the loop completion. Does this make sense to do the same here as well?

Reply:
Latest commit fixes this. I also scanned the whole PR for any similar instances, but this seems to be the only one.